src/BinaryParsers/PEBinary/ProgramDatabase/CompilerCommandLine.cs

```diff
@@ -74,6 +79,8 @@ public CompilerCommandLine(string commandLine)
             this.WarningLevel = 0;
             this.WarningsAsErrors = false;
             this.OptimizationsEnabled = false;
+            this.UsesDebugCRuntime = false;
+
             var explicitWarnings = new Dictionary<int, WarningState>();
             foreach (string argument in ArgumentSplitter.CommandLineToArgvW(commandLine))
             {
```

Comment:
line 92 - why bother switching on length to subdivide the ifs?

Reply:
This is a preexisting pattern so I can only speculate.  

The compiler can get both `-MT` and `/MT` as valid options so using `EndsWith("MT")` will work regardless of whether a - or / is used.  You don't want to over-match to other strings so requiring the length to match expectations will make this safe.